docker/client.py

```diff
@@ -747,6 +749,10 @@ def inspect_container(self, container):
 
     @check_resource
     def inspect_image(self, image):
+        if not image:
+            raise ValueError('image or ID required')
+        if isinstance(image, dict):
```

Comment:
This wasn't part of the original issue, but is included to provide consistency with the inspect_container() method.